Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2538,6 +2539,16 @@ def lhegensim2018ml(fragment,howMuch):
                                        '-s':'RECO:bunchSpacingProducer+ecalMultiFitUncalibRecHitTask+ecalCalibratedRecHitTask',
                                        '--customise':'Calibration/EcalCalibAlgos/EcalPhiSymRecoSequence_cff'},dataReco])
 
+steps['NanoRun3ScoutingPF']=merge([{'-s':'NANO',
+                         '--conditions':'auto:run3_data',
+                         '-n':'10',
+                         '--era' : 'Run3',
+                         '--geometry' : 'DB:Extended',
+                         '--datatier':'NANOAOD',
+                         '--eventcontent':'NANOAOD',
+                         '--customise':'"PhysicsTools/NanoAOD/custom_run3scouting_cff.PrepRun3ScoutingCustomNanoAOD_Data"',
```

Comment:
can you please work out a configuration that would allow `--step NANO:<path to cff>.<sequence name>` or `--step NANO:<path to cff>`